challenge_storing_groceries/src/challenge_storing_groceries/inspect_shelves.py

```diff
@@ -15,11 +15,12 @@
 class InspectShelves(smach.State):
     """ Inspect all object shelves """
 
-    def __init__(self, robot, cabinet):
+    def __init__(self, robot, cabinet,inspection_result_designator):
         smach.State.__init__(self, outcomes=['succeeded', 'failed', 'nothing_found'])
         self.robot = robot
         self.cabinet = cabinet
         self.object_shelves = config.OBJECT_SHELVES
+        self.inspection_result_designator = inspection_result_designator
```

Comment:
Nice. In general, it's indeed *much* nicer to pass a container to store the results than to store them globally in the config module